gcsfuse/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+
+- name: Add gcsfuse repo
+  apt_repository: repo="{{ gcsfuse_repo }}"
+  tags: ['gcsfuse', 'gcsfuse:install']
+
+- name: Add repository key
+  apt_key: url="{{ gcsfuse_apt_key_url }}"
+  tags: ['gcsfuse', 'gcsfuse:install']
+
+- name: Install gcsfuse
+  apt: name="{{ gcsfuse_package }}" update_cache=yes
+  tags: ['gcsfuse', 'gcsfuse:install']
+
+- name: Create mount directories
+  file:
+    state: directory
+    path: "{{ item.path }}"
+    owner: "{{ item.owner | default(omit) }}"
+    group: "{{ item.group | default(omit) }}"
+    mode: "{{ item.mode | default(omit) }}"
+  become: True
+  become_user: "{{ item.owner | default('root') }}"
+  with_items: "{{ gcsfuse_mounts }}"
+  tags: ['gcsfuse', 'gcsfuse:configuration']
+
+- name: Create directories for credentials files
+  file:
+    state: directory
+    path: "{{ item.dest | dirname }}"
+  with_items: "{{ gcsfuse_keyfiles }}"
+  tags: ['gcsfuse', 'gcsfuse:configuration']
+
+- name: Copy credentials files for service accounts
+  copy:
+    content: "{{ item.content | b64decode }}"
+    dest: "{{ item.dest }}"
+    owner: "{{ item.owner | default(omit) }}"
+    group: "{{ item.group | default(omit) }}"
+    mode: "{{ item.mode | default(omit) }}"
+  with_items: "{{ gcsfuse_keyfiles }}"
+  tags: ['gcsfuse', 'gcsfuse:configuration']
+
+- name: Mount buckets in directories
+  shell: gcsfuse {{ (item.keyfile != None) | ternary("--key-file " + item.keyfile, "") }} "{{ item.bucket }}" "{{ item.path }}" && touch "{{ item.path }}/.mounted"
```

Comment:
I'm not asking to change this, but curious, would breaking this into multiple lines break it?